modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -120,9 +128,76 @@ public void testAddUserWithInvalidName() {
 		}
 	}
 
+	@Test
+	public void testGetRoleUserIdsWithNoPermissions() throws Exception {
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		User user1 = UserTestUtil.addUser();
+
+		try {
+			Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+			_userLocalService.addRoleUser(role.getRoleId(), role.getUserId());
```

Comment:
mmm... don't we have to add the role to user1?

Reply:
@smottal , it was small detail I missed :sweat_smile: . I messed up when separating the test into two tests (noPermissions vs. permissions) when I was originally reusing the patterns for each method.